memtier_benchmark.cpp

```diff
@@ -903,6 +948,7 @@ void usage() {
             "      --key=FILE                 Use specified private key for TLS\n"
             "      --cacert=FILE              Use specified CA certs bundle for TLS\n"
             "      --tls-skip-verify          Skip verification of server certificate\n"
+            "      --tls-protocols            Specify the tls protocol version to use, comma delemited. Use a combination of 'TLSv1', 'TLSv1.1', 'TLSv1.2' and 'TLSv1.3'"
```

Comment:
Doesn't `--tls-versions` will be more correct?

Reply:
@YaacovHazan i've tried to keep the same config name from redis. https://app.circleci.com/pipelines/github/RedisGraph/RedisGraph/11092/workflows/1c9a1a6b-88f1-45e7-9fa4-fea295d41a4a/jobs/49613/parallel-runs/0/steps/0-113